wordpress-mobile/WordPress-iOS
# Update stats pod to 0.7.0 and fix HockeySDK

Fixes #5151 
Related #5146

Also fixes a bad merge in #5146 for the HockeyApp SDK.

To test:
1. Run the app with a .com login.
2. Load stats.
3. Ensure insights posting trends renders.

Needs review: @bummytime
